crates/mdk-core/src/encrypted_media/manager.rs

```diff
@@ -70,7 +70,11 @@ where
         filename: &str,
         options: &MediaProcessingOptions,
     ) -> Result<EncryptedMediaUpload, EncryptedMediaError> {
-        let canonical_mime_type = validate_inputs(data, mime_type, filename, options)?;
+        // Validate inputs
+        let validation_options = options.to_image_validation_options();
```

Comment:
Same here regarding the wording, maybe rename this to `to_media_validation_options` or `to_validation_options`?